### PR TITLE
Gometalinter rule

### PR DESCRIPTION
**Pull request by ash2k**

### CODE REVIEW

File: gometalinter/deps.bzl [hunk 1]
```diff
@@ -0,0 +1,50 @@
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+
+_GOMETALINTER_TARBALLS = {
+    "darwin_amd64": ("gometalinter-2.0.5-darwin-amd64.tar.gz", "gometalinter-2.0.5-darwin-amd64", "d2571dd081a00752fc7876a9f64042937e9d75e7968465e3c5b5ee9dad64a6c7"),
```
Comment (wryun) [inline gometalinter/deps.bzl]: there's no nicer way than this? We can't set the rule up to depend on a version, with a default of 2.0.5?

Comment (ash2k) [reply gometalinter/deps.bzl]: You mean make the version configurable? It is doable, but I don't think we need it. That 2.0.5 version was released in February. They don't release often, so we are going to be up to date for a while most likely.
I was just copying from rules_go
https://github.com/bazelbuild/rules_go/blob/ca5ffac455080a3b7c2ecd8cdbbb748c9477411d/go/toolchain/toolchains.bzl#L25
https://github.com/bazelbuild/rules_go/blob/238329474a90d8ac8c128cec3184c515ef467c04/go/private/sdk.bzl#L28-L57

**Comment by ash2k** [comment]
I'd expect that, but I don't know when the broader community will start adopting modules. We'll see. Yes, GOPATH is going away IIUC. I haven't been closely following the vgo/modules story.

File: gometalinter/deps.bzl (hunk 1 above)
Comment (amckague) [reply gometalinter/deps.bzl]: When we start using this we'll be rolling back from master to February. :man_shrugging:

**Pull request merged**